SYCL/Basic/device_code_dae.cpp

```diff
@@ -2,8 +2,10 @@
 // driver
 // UNSUPPORTED: cuda
 // CUDA does not support SPIR-V.
-// RUN: %clangxx -fsycl-device-only -Xclang -fenable-sycl-dae -Xclang -fsycl-int-header=int_header.h %s -c -o device_code.bc -Wno-sycl-strict
-// RUN: %clangxx %cxx_std_optionc++17 %include_option int_header.h %debug_option -c %s -o host_code.o %sycl_options -Wno-sycl-strict
+// FIXME Disabled fallback assert as it'll require either online linking or
+// explicit offline linking step here
```

Comment:
Why not to add explicit offline linking like in previous test?

Reply:
Per offline discussion, agreed to add offline linking to this test in distinct patch.